### PR TITLE
Add tests for MaxByteOffset.

### PR DESCRIPTION
**Pull request by dekimir**
@antiagainst @AWoloszyn @dneto0 @Qining PTAL.

### CODE REVIEW

File: test/TextToBinary.Annotation.cpp [hunk 1]
```diff
@@ -394,70 +410,80 @@ TEST_F(TextToBinaryTest, GroupMemberDecorateInvalidSecondTargetMemberNumber) {
...
+                {CASE(InputAttachmentIndex), {102}},
+                {CASE(Alignment), {16}},
+            })), );
 #undef CASE
 
+INSTANTIATE_TEST_CASE_P(
+    TextToBinaryDecorateSimpleV11, OpMemberDecorateSimpleTest,
+    Combine(Values(SPV_ENV_UNIVERSAL_1_1),
+            Values(EnumCase<SpvDecoration>{
+                SpvDecorationMaxByteOffset, "MaxByteOffset", {128}})), );
```
Comment (antiagainst) [inline test/TextToBinary.Annotation.cpp]: Why not using `CASE` here, too?

File: test/TextToBinary.Annotation.cpp [hunk 2]
```diff
@@ -30,85 +30,101 @@
...
+                {CASE(InputAttachmentIndex), {102}},
+                {CASE(Alignment), {16}},
+            })), );
 #undef CASE
 
+INSTANTIATE_TEST_CASE_P(
+    TextToBinaryDecorateSimpleV11, OpDecorateSimpleTest,
+    Combine(Values(SPV_ENV_UNIVERSAL_1_1),
+            Values(EnumCase<SpvDecoration>{
+                SpvDecorationMaxByteOffset, "MaxByteOffset", {128}})), );
```
Comment (antiagainst) [inline test/TextToBinary.Annotation.cpp]: Use `CASE` here?

**Comment by antiagainst** [comment]
+1

two nitpicks I don't really care.

**Comment by dekimir** [comment]
Addressed all nitpicks. 😄

**Comment by antiagainst** [comment]
LGTM

**Pull request merged**